mlops_utilities/actions.py

```diff
@@ -7,21 +7,22 @@
 
 import boto3
 from omegaconf import OmegaConf
-from sagemaker import ModelPackage, Predictor, Session
+from sagemaker import ModelPackage, Predictor
 from sagemaker.model_monitor import DataCaptureConfig
+from sagemaker.workflow.pipeline_context import PipelineSession
 
 from mlops_utilities import helpers
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger(__name__).addHandler(logging.NullHandler())
 
 
 def upsert_pipeline(
-    *args,
     pipeline_module: str,
     pipeline_package: str,
     pipeline_name: str,
     config_type: str,
     role: str,
+    *args,
```

Comment:
🙂👍

Reply:
Linter firstly confused me, but now I clearly understand right place for *